app.js

```diff
@@ -0,0 +1,16 @@
+const express = require("express");
+const app = express();
+const { getTopics } = require("./controllers");
+
+app.get("/api/topics", getTopics);
+
+app.use((err, req, res, next) => {
+		res.status(err.status).send({ message: err.message });
+	next(err);
+});
+
+app.all("*", (request, response, next) => {
```

Comment:
This is the end of the express chain, so should be below all express endpoints, but above your error handling middleware